Lib/warnings.py

```diff
@@ -7,6 +7,8 @@
 import sys
 import types
 
+# sys.setrecursionlimit(1 << 30)
```

Comment:
Is this supposed to be `if sys.py3kwarning: sys.setrecursionlimit(...)`?

Reply:
I used` if not sys.py3kwarning` because that was the default limit that I was overriding. See https://github.com/softdevteam/pygrate2/pull/36/commits/85cde3704f91f2f4d5f7e438346297f27563f40c